ddtrace/internal/forksafe.py

```diff
@@ -46,6 +48,7 @@ def register(after_in_child):
     _threading_after_fork = threading._after_fork
 
     def _after_fork():
+        # type: () -> None
         _threading_after_fork()
         ddtrace_after_in_child()
```

Comment:
~~~ python
else:
    import threading

    register(threading._after_fork)
    threading._after_fork = ddtrace_after_in_child
~~~

Reply:
**NOTE**: Doesn't work with `set`s if order matters.

Reply:
The original `_after_fork` has nothing to do with what is registered so having it outside sounds better to me.
e.g. we don't want to allow it to be unregistered